topics/programming-language/index.md

```diff
@@ -0,0 +1,7 @@
+---
+display_name: Programming language
+short_description: A programming language is a language for developing apps and all other sorts of programs.
```

Comment:
I think we could be a bit more descriptive about how a programming language is something that can be used to write instructions for a computer to interpret, or something along those lines.